docs/content/learn/tutorials/component/_index.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Creating a Component Definition
+description: A tutorial on creating an OSCAL component definition.
+weight: 6
+suppresstopiclist: true
+toc:
+  enabled: true
+aliases:
+  - /tutorials/component/
+---
+
+This tutorial covers creating a basic OSCAL component definition. Before reading this tutorial you should:
+
+- Have some familiarity with the [XML](https://www.w3.org/standards/xml/core), [JSON](https://www.json.org/), or [YAML](https://yaml.org/spec/) formats.
+- Read the OSCAL implementation layer [overview](/documentation/schema/implementation-layer/).
+- Review the OSCAL [component definition model documentation](/documentation/schema/implementation-layer/component/).
+- Be familiar with OSCAL properties and links.
+
+## What is an OSCAL Component Definition?
+
+A component definition describes how a given implementation of a component or sets of components (e.g., hardware, software, service, policy, process, or procedure) can support specific controls.  Product and Service Vendors, Capability Owners, Policy and Process Owners, and others can create component definitions that readily serve as inputs for a System Security Plan (SSP), helping describe how components might satisfy controls.  It is important to note that component definitions do not describe actual implementations. Component definitions serve as references with content that can be used (e.g., in the SSP OSCAL model) to develop comprehensive and consistent control implementations within an SSP.
+
+In this tutorial, we will walk through the process of creating an [OSCAL component definition model](/documentation/schema/implementation-layer/component/) for MongoDB.  Our goal is to provide standard control implementations for the benefit of system owners and SSP authors. As potential components in systems that may need to meet the OMB A-130 Authorization to Operate requirements, the component definition in this tutorial will demonstrate proper implementation of a couple [NIST SP 800-53 rev 5 controls](https://github.com/usnistgov/oscal-content/tree/master/nist.gov/SP800-53/rev5/xml) deemed necessary for high / moderate / low impact systems. While this example focuses on NIST controls, the same approach could be used for any cybersecurity frameworks and their respective controls.  We'll presume the MongoDB component can partially or fully satisfy the following controls:
```

Comment:
A hyperlink to the MongoDB product's page could be useful. (e.g https://docs.mongodb.com/)